From 4b3d3fc4e2e489706a1a4ad3ced1ca5cc81cb583 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 17 Oct 2017 17:49:48 +0200
Subject: [PATCH] test-path: fix inverted return value (#7050)

It must be <0 to indicate failure.

Fixes #7050.

(cherry picked from commit be21bf856b77629c7f49ca65b3e62f650097048a)
---
 src/test/test-path.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/test-path.c b/src/test/test-path.c
index 9de3dd6674..051dfc4868 100644
--- a/src/test/test-path.c
+++ b/src/test/test-path.c
@@ -48,7 +48,7 @@ static int setup_test(Manager **m) {
         r = enter_cgroup_subroot();
         if (r == -ENOMEDIUM) {
                 log_notice_errno(r, "Skipping test: cgroupfs not available");
-                return EXIT_TEST_SKIP;
+                return -EXIT_TEST_SKIP;
         }
 
         r = manager_new(UNIT_FILE_USER, MANAGER_TEST_RUN_MINIMAL, &tmp);
